jsapp/js/components/processing/singleProcessingStore.ts

```diff
@@ -30,12 +25,12 @@ import type {
 import type {LanguageCode} from 'js/components/languages/languagesStore';
 import type {AnyRowTypeName} from 'js/constants';
 import {destroyConfirm} from 'js/alertify';
+import {
+  isAnyProcessingRouteActive,
+  getProcessingPathParts,
+} from 'js/components/processing/routes.utils';
 
-export enum SingleProcessingTabs {
-  Transcript = 'trc',
-  Translations = 'trl',
-  Analysis = 'an',
-}
+export type ProcessingTabName = 'transcript' | 'translations' | 'analysis';
```

Comment:
This should stay an enum - that way you won't have to use these strings in `DefaultDisplays`, `getActiveTab()`, `onRouteChange()`, and `cleanUpDisplays()`.

Reply:
When it was an `enum`, I had to use `ProcessingTabName.Translations`. With `type` I can use shorter `'translations'`. With both of these the type safety is similar, right? I was using `enum`s a lot few years back, but I found out that while they are more "correct" theoretically speaking, a simple type is more concise.

I guess we should have a small discussion which way do we prefer?

Reply:
I brought back `enum` :)

Reply:
Thought I could make that `enum` use routes strings and avoid using that `TabToRouteMap`, but turns out you can compute only numbers :shrug: